### PR TITLE
net-im/profanity: version bump to 0.12.1

### PR DESCRIPTION
**Pull request by mrusme**

### CODE REVIEW

File: net-im/profanity/profanity-0.12.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (Flowdalic) [inline net-im/profanity/profanity-0.12.1-r1.ebuild:4]: EAPI 8?

File: net-im/profanity/Manifest [hunk 2]
```diff
@@ -1,2 +1,3 @@
-DIST profanity-0.9.5.tar.gz 830966 BLAKE2B fc1b755b1a92dbde62aacb31cb5d9190cea951e98501a5b48495dbd31266f343cf436de1fa0f69caec34324cd1f781d0bd4f5f7ebdd984324c5d1d6edb91452a SHA512 79305ee488d3a9b87bb422d6525bfaae62c4969d466b968c86941ed5bf0c0080e349758f2e1ead144693513b5d19a635372e73aa1fe9e362836b9245e1d2b898
```
Comment (Flowdalic) [inline net-im/profanity/Manifest:1]: Unnecessary change of ordering

**Review by Flowdalic — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please fix the missing signoff. Then I'll have a closer look at it.

**Comment by mrusme** [comment]
@Flowdalic thank you for the feedback, kindly re-check!

**Comment by Flowdalic** [comment]
> @Flowdalic thank you for the feedback, kindly re-check!

The 'Signed-off-by:' is on the same line as the commit subject. See https://github.com/gentoo/gentoo/commit/f3ac9ad393dd879ff89174a1ab4bad34596a401f.patch

Please fix this.

**Pull request closed without merging**